src/api/v1/user/index.js

```diff
@@ -3,6 +3,7 @@ const express = require('express');
 const error = require('../../../error');
 const { User, Activity, db } = require('../../../db');
 const Storage = require('../../../storage');
+const Media = require('../../../storage/media');
```

Comment:
Nit: `Media` is part of `Storage`, so keep the imports structured as such, e.g. `Storage.mediaTypes.PROFILE_PICTURE`.

Reply:
I don't understand what you mean by this, are you saying I should have the media types/functions not in a separate file, but under the `storage/index.js`? Because I was thinking abstracting that away from the `storage/index.js` into a `storage/media.js` file makes that index file a lot less cluttered and structured

Reply:
That's fine, just re-export the structs so everything storage-related (especially the enum) can be accessed through the `Storage` import.